src/engine/controls/vinylcontrolcontrol.cpp

```diff
@@ -89,21 +89,15 @@ void VinylControlControl::slotControlVinylSeek(double fractionalPos) {
         return;
     }
 
-    const auto trackEndPosition =
-            mixxx::audio::FramePos::fromEngineSamplePosMaybeInvalid(
-                    getSampleOfTrack().total);
-    if (!trackEndPosition.isValid()) {
-        return;
-    }
     const auto newPlayPos = mixxx::audio::kStartFramePos +
```

Comment:
This might assert after removing the conditional. Or is this case unexpected and we don't need to check for it?

Reply:
I'm no 100% sure, but intuitively this can only be invalid if no track is loaded. But maybe this is prone to data races. I guess I should retrieve the frame info at the start of the method and add the isValid check where we check if no track is loaded.